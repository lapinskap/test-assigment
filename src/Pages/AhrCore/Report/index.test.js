import React from 'react';
import renderer from 'react-test-renderer';
import ReportRouter from './index';
import '../../../tests/setupTests';
import TestAppContext from '../../../tests/TestAppContext';

describe('<ReportRouter />', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  it('renders correct snapshot with simple props', () => {
    const tree = renderer
      .create(
        <TestAppContext router>
          <ReportRouter match={{ url: '/ahr/report', params: {}, path: '' }} />
        </TestAppContext>,
      )
      .toJSON();
    expect(tree)
      .toMatchSnapshot();
  });
});
