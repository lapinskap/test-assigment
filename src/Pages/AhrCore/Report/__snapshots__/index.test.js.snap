// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReportRouter /> renders correct snapshot with simple props 1`] = `null`;
